protobuf/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## 3.0.1-dev
 
+* Avoid holding onto the input buffer when parsing unknown length-delimited
```

Comment:
Would it make sense to publish it as `3.0.1` or `3.0.0+1`?

Reply:
It makes sense to publish this, yes. With the renaming we'll need a major version bump.

Reply:
It makes sense to publish this, yes. With the renaming we'll need a major version bump.